static-crawling/static-crawling_assignment.py

```
@@ -0,0 +1,176 @@
+"""
+페이지를 변경하면서 여러 페이지의 데이터 크롤링을 해봅시다.
+
+- for page_num in range(1, 4): 1페이지부터 3페이지까지 순차적으로 크롤링합니다.
+2. 내부 for loop에서 위 코드에서 실행했던 한 페이지에서 도서 정보 모으기를 실행합니다.
+3. DataFrame에 정보를 저장합니다.
+"""
+
+datas = []
+for page_num in range(1, 4):
+    url = f"https://www.aladin.co.kr/shop/common/wbest.aspx?BestType=Bestseller&BranchType=1&CID=0&page={page_num}" 
+    response = requests.get(url)  
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)